src/app/frontend/resource/workloads/job/detail/template.html

```diff
@@ -71,6 +71,9 @@
   </div>
 </kd-card>
 
+<kd-condition-list [conditions]="job?.jobStatus.conditions"
```

Comment:
It would be safe to put ? also before conditions.

Reply:
Good catch.
I adjusted to `job?.jobStatus?.conditions`.